### ISSUE TITLE
shared-grive caching issue

### ISSUE THREAD

**Issue by sorawee** [brownplt/code.pyret.org#227]
Say, I have two Pyret code open: `a.arr` and `b.arr` where `a.arr` imports `b.arr`. I run `a.arr` and find that `b.arr` is wrong, so I edit and republish `b.arr`. When I run `a.arr` again, it still runs the old `b.arr`. I need to refresh the page (start Pyret again) of `a.arr` to update to the latest version of `b.arr`.

This will really affect students who have implementations and tests on two tabs.

I understand the need of caching, but can it be less aggressive. I.e., look at the timestamp of shared code. If it's newer than what gets run before, then clear the cache and reload the file.

**Comment by jpolitz** [comment]
Can't students use "my-gdrive" for this use case? It should do the right file-based caching for this.

**Comment by schanzer** [comment]
@sorawee is this still an issue on 2025's CPO?

**Comment by schanzer** [comment]
@sorawee poke?

**Comment by jpolitz** [comment]
I can confirm this is still an issue. However, I'm going to close it as wontfix for now. We have a _much_ better solution for this development case with `url-file`, and I think that people working on multi-file projects should be guided away from CPO and towards a better supported multi-file environment. Curriculum developers who naturally work on multiple files should start moving towards `url-file` as well to avoid this annoyance.

**Comment by schanzer** [comment]
@jpolitz with `url-file` up and running, and you offering to close this a year ago....OK to close this now?

**Comment by jpolitz** [comment]
Yeah we have a better solution for this now (the issue is still present, but it's OK to be "wontfix")
